src/platform/silabs/rs911x/wfx_sl_ble_init.c

```diff
@@ -184,8 +187,13 @@ int32_t rsi_ble_app_get_event(void)
  */
 void rsi_ble_app_set_event(uint32_t event_num)
 {
+    SILABS_LOG("%s: starting", __func__);
     event_msg.ble_app_event_map |= BIT(event_num);
+#ifndef SI917	
     rsi_semaphore_post(&sl_ble_event_sem);
+#else
+    osSemaphoreRelease(sl_ble_event_sem);

```

Comment:
same comment as before. No real reason to keep the rsi_semaphore calls and have if defs. Let's just use CMSIS calls for all if you start using it.

Reply:
done @jmartinez-silabs

Reply:
done